src/ownership/move-semantics.md

```diff
@@ -12,15 +12,18 @@ fn main() {
 ```
 
 * The assignment of `s1` to `s2` transfers ownership.
-* The data was _moved_ from `s1` and `s1` is no longer accessible.
-* When `s1` goes out of scope, nothing happens: it has no ownership.
+* When `s1` goes out of scope, nothing happens: it does not own anything.
 * When `s2` goes out of scope, the string data is freed.
 * There is always _exactly_ one variable binding which owns a value.
 
 <details>
 
 * Mention that this is the opposite of the defaults in C++, which copies by value unless you use `std::move` (and the move constructor is defined!).
 
+* It is only the ownership that moves. Whether the resulting machine code copies any data around is a matter of optimization, and such copies are aggressively optimized away.
```

Comment:
The copy here is confusing again, right? 😬

Reply:
Maybe this, although "manipulate" feels a bit awkward.
```suggestion
* It is only the ownership that moves. Whether any machine code is generated to manipulate the data itself is a matter of optimization, and such copies are aggressively optimized away.
```

Reply:
I like it better